src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -29,10 +29,12 @@ public class DocumentExecuter : IDocumentExecuter
     {
         private readonly IDocumentBuilder _documentBuilder;
         private readonly IDocumentValidator _documentValidator;
+        public bool Strict { get; set; }
```

Comment:
I can understand what you want here, though would Validation perhaps solve the problem you're looking to solve?  Is that just essentially validation?  I don't want to move too far away from the official spec on behavior.